includes/admin/downloads/metabox.php

```diff
@@ -923,29 +937,33 @@ function edd_render_refund_row( $post_id ) {
 	$edd_refund_window = edd_get_download_refund_window( $post_id ); ?>
 
 	<div class="edd-product-options-wrapper">
-		<p><strong><?php _e( 'Refundability', 'easy-digital-downloads' ); ?><span alt="f223" class="edd-help-tip dashicons dashicons-editor-help" title="<?php _e( '<strong>Refundability</strong>: Allow or disallow refunds for this specific product. When allowed, the refund window will be used on all future purchases.', 'easy-digital-downloads' ); ?>"></span></strong></p>
-		<p><?php echo EDD()->html->select( array(
-			'name'     => '_edd_refundability',
-			'options'  => $types,
-			'selected' => $edd_refundability,
-			'chosen'   => true,
+		<p>
+			<strong>
+				<?php esc_html_e( 'Refunds', 'easy-digital-downloads' ); ?>
+				<span alt="f223" class="edd-help-tip dashicons dashicons-editor-help" title="<?php echo wp_kses( __( '<strong>Refundability</strong>: Allow or disallow refunds for this specific product. When allowed, the refund window will be used on all future purchases.<br /><strong>Refund Window</strong>: Limit the number of days this product can be refunded after purchasing.', 'easy-digital-downloads' ), array( 'strong' => true, 'br' => true ) ); ?>"></span>
```

Comment:
Maybe `Refundability` in the tooltip should be renamed to `Refundable` since you removed the old `Refundability` heading.